### ISSUE TITLE
Section 2.2.2 - reference on "automatisable" tests (Karl Groves)

### ISSUE THREAD

**Issue by darthcav** [w3c/w3c-waet#11]
## Reference

http://lists.w3.org/Archives/Public/public-wai-ert-tools/2014Jul/0005
## Original Comment

This section includes the claim “Since it is a known fact that automatic tests only cover a small set of accessibility issues...”.  While I agree with this statement, it’d be good to have a reference cited in this section. It would also be prudent to disclose the likely disparity between what can theoretically be tested versus what is commonly tested.

**Comment by nitedog** [comment]
## Resolution

Partial - mentioned in rewording of section 2.2.2. I think "it is a known fact that..." is problematic in many ways and invites exactly that type of challenge. It is also too wordy. We should rewrite this section to avoid such statements altogether. See also Issue #45

**Comment by darthcav** [comment]
See new wording.
